MiauAPI/Models/QueryObjects/ProductReviewObject.cs

```diff
@@ -3,22 +3,18 @@
 namespace MiauAPI.Models.QueryObjects;
 
 /// <summary>
-/// Represents the object in a product query.
+/// Represents the object in a product review query.
 /// </summary>
-/// <param name="Id">The description of the product.</param>
-/// <param name="Name">The name of the product.</param>
-/// <param name="Description">The description of the product.</param>
-/// <param name="Price">The price of the product.</param>
-/// <param name="IsActive">Determines whether this product is available for sale.</param>
-/// <param name="Amount">The product's amount in stock.</param>
-/// <param name="Tags">The product's store tags.</param>
-/// <param name="Brand">The product's brand.</param>
-/// <param name="Discount">The password entered by the user.</param>
+/// <param name="Id">The id of the user review.</param>
+/// <param name="UserId">The id of the user.</param>
+/// <param name="ProductId">The id of the product.</param>
+/// <param name="Description">Description of the review.</param>
+/// <param name="Score">The product score.</param>
 public sealed record ProductReviewObject
 {
     public int Id { get; set; }
-    public DateTime DateAdded { get; set; }
     public int UserId { get; set; }
     public int ProductId { get; set; }
+    public string? Description { get; set; }
     public int Score { get; set; }
-} 
\ No newline at end of file
+}
```

Comment:
Você pode reescrever esse `record` com [sintaxe posicional](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record#positional-syntax-for-property-definition).